iOS-Play/iOS-Play/Controller/AppViewController/NewGameViewController.swift

```diff
@@ -8,16 +8,153 @@
 import Foundation
 import UIKit
 
-class NewGameViewController : UIViewController {
+final class NewGameViewController: UIViewController {
+    
+    private let tableView = UITableView(frame: .zero, style: .plain).then {
+        $0.register(cellType: UITableViewCell.self)
+        $0.register(cellType: CollectionTableViewCell.self)
+        $0.register(cellType: GenreViewCell.self)
+        $0.register(cellType: HighlighTableViewCell.self)
+        $0.register(cellType: BanerViewTableCell.self)
+        $0.showsVerticalScrollIndicator = false
+        $0.showsHorizontalScrollIndicator = false
+        $0.separatorColor = .clear
+    }
+    
+    private let dataTest = DataTest.shared
+    
+    private var models = [CellModel]()
+    
+    private let heightOfGenre: CGFloat = 30
+    private let heightOfCollection: CGFloat = 200
+    private let heightOfHighlight: CGFloat = 350
+    private let heightOfBaner: CGFloat = 310
+    private let heightOfList: CGFloat = 50
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         configViews()
+        //configCellModell()
+        models = dataTest.getDataTest()
     }
     
     func configViews() {
-        view.backgroundColor = .systemTeal
+        view.backgroundColor = .white
+        
+        tableView.do {
+            $0.delegate = self
+            $0.dataSource = self
+        }
+        
+        view.addSubview(tableView)
+        
+        tableView.do {
+            $0.snp.makeConstraints { (make) in
+                make.bottom.left.top.right.equalToSuperview()
+            }
+        }
     }
     
+    func configCellModell() {
+        models = dataTest.getDataTest()
+    }
+}
+
+
+extension NewGameViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return models.count
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch models[section] {
+        case .genreTitle: return 1
+        case .collectionView: return 1
+        case .highlightView: return 1
+        case .banerView: return 1
+        case .listView(model: let models): return models.count
```

Comment:
```suggestion
        case .listView: return models.count
```

Reply:
em dùng models.count trong listView nên phải có "model: let models" ạ :vvv

Reply:
thế thì đổi thành như này
```suggestion
        case .listView(let models): return models.count
```